includes/class-wc-order-item-product.php

```diff
@@ -167,7 +167,7 @@ public function set_taxes( $raw_tax_data ) {
 	 */
 	public function set_variation( $data ) {
 		foreach ( $data as $key => $value ) {
-			$this->add_meta_data( str_replace( 'attribute_', '', $key ), $value, true );
```

Comment:
https://github.com/woocommerce/woocommerce/blob/2.6.14/includes/abstracts/abstract-wc-order.php#L301 It looks like 2.6 removes the prefix. I'll modify this to preserve that same functionality.